pkg/updater/genesis_manifests.go

```diff
@@ -51,16 +44,30 @@ func getRelevantDownloadURL(centralEndpoint string) (string, error) {
 	if len(manifest.KnownGenesisDumps) == 0 {
 		return "", errors.New("invalid manifest, no genesis dumps")
 	}
-
 	var mostRecentGenesisDump *knownGenesisDump
 	for i, dump := range manifest.KnownGenesisDumps {
 		if mostRecentGenesisDump == nil || dump.Timestamp.After(mostRecentGenesisDump.Timestamp) {
 			mostRecentGenesisDump = &manifest.KnownGenesisDumps[i]
 		}
 	}
-
 	uuid := mostRecentGenesisDump.UUID
 	if err := validateUUID(uuid); err != nil {
+		return "", err
+	}
+	return uuid, err
+}
+
+// getRelevantDownloadURL gets the genesis manifests from the dump, finds the one
+// with the highest timestamp, and returns the location for the diff dump from that location.
+// This ensures that we get the smallest diff dump that works for this version of scanner.
+func getRelevantDownloadURL(centralEndpoint string) (string, error) {
```

Comment:
is this solely used for Central communication? If so, let's try to specify that in the name

Reply:
In case you missed this ^

Reply:
That was also copied and pasted. When moving things like, I think it's better for everyone to not modify the code and make it easier on future readers that this was a C'n'P. We can refactor the name in a different PR.